modules/benchmarks-cs/ia_loop.cs

```diff
@@ -0,0 +1,372 @@
+using SpacetimeDB;
+
+namespace Benchmarks;
+
+public static partial class ia_loop
+{
+    [SpacetimeDB.Table]
+    public partial struct Velocity(uint entity_id, float x, float y, float z)
+    {
+        [SpacetimeDB.Column(ColumnAttrs.PrimaryKey)]
+        public uint entity_id = entity_id;
+        public float x = x;
+        public float y = y;
+        public float z = z;
+    }
+
+    [SpacetimeDB.Table]
+    public partial struct Position(uint entity_id, float x, float y, float z)
+    {
+        [SpacetimeDB.Column(ColumnAttrs.PrimaryKey)]
+        public uint entity_id = entity_id;
+        public float x = x;
+        public float y = y;
+        public float z = z;
+        public float vx = x + 10.0f;
+        public float vy = y + 20.0f;
+        public float vz = z + 30.0f;
+    }
+
+    public static ulong MomentMilliseconds()
+    {
+        // Idk why Rust uses complicated math, but it seems like it should always return 1.
+        return 1;
```

Comment:
It was purely to copy the way was done and use the time methods, but is likely that this could be just a `const`